### ISSUE TITLE
VSCode.dev compatibility

### ISSUE THREAD

**Issue by sancarn** [serkonda7/vscode-vba#105]
### Description and Reproduction Steps

For some reason, I'm yet to figure out totally why, but the extension isn't installable on http://vscode.dev ...

Looking [here](https://code.visualstudio.com/api/extension-guides/web-extensions#web-extension-anatomy) there could be many culprits e.g.

> For web extensions, the [main entry file](https://code.visualstudio.com/api/extension-guides/web-extensions#web-extension-main-file) is defined by the **`browser`** property, and not by the `main` property as with regular extensions.

**Comment by serkonda7** [comment]
Yeah, we do not define the `browser` property. 
I'm not fully sure however if it will work without some more infrastructure work from my side.

**Comment by sancarn** [comment]
I'm not 100% sure either. Looking through the project though it looks like adding a browser property would be sufficient. However it's difficult to know for sure without testing it.

**Comment by DecimalTurn** [comment]
Related: https://code.visualstudio.com/docs/editor/vscode-web#_extensions

**Comment by serkonda7** [comment]
Thanks for the link. I will work on it once I finish vba decompression (which is making progress)

**Comment by sancarn** [comment]
> Thanks for the link. I will work on it once I finish vba decompression (which is making progress)

VBA decompression? Interested what you mean by that... Is that a typescript rewrite of Beakerboy's python lib?

**Comment by serkonda7** [comment]
Kinda. Rewrite in a language that compiles down zo C in case performance is needed or to Javascript for compability.
Also it's technically not a rewrite as I take a slightly different approach.

**Comment by DecimalTurn** [comment]
For info, I tried to simply add `"browser": "./out/extension.js",` to package.json and it worked when I tested it using the [sideloading approach](https://code.visualstudio.com/api/extension-guides/web-extensions#test-your-web-extension-in-vscode.dev) and was able to install the extension on http://vscode.dev/:

![image](https://github.com/serkonda7/vscode-vba/assets/31558169/8d6f61a8-69e7-4c6f-9fd2-e889e7cf8457)
(There is an error related to package.nls.json`, but it doesn't stop the extension from being loaded. Something to do with translating the extension it seems).

I was also able to use the syntax highlighting and insert snippets without any issues. It seems that until more features are added, the changes required to make the extension work online should be minimal.
